src/components/icons/microscope.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+export const MicroscopeIcon = () =>
+  <svg viewBox="-15 -10 90 130" className="expedition-group-icon">
```

Comment:
I think the microscope svg sits a little high? I'm not experienced with svgs like I'd like to be, but changing the svg viewBox attributes' second number from `-10` to `-25` or `-30` looked ok in Chrome.